### PR TITLE
Relative pathing

### PR DESCRIPTION
**Pull request by mskrip**
This enables executing the built executable from any working directory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,27 +12,20 @@ setup instructions check the [github](https://github.com/faiface/pixel).
 
 ## Building and running
 
-First you need to install dependencies:
+First you need to install dependencies build the project and run with:
```
Comment (mitas1) [inline README.md]: missing comma

File: README.md [hunk 2]
```diff
@@ -12,27 +12,20 @@ setup instructions check the [github](https://github.com/faiface/pixel).
 
 ## Building and running
 
-First you need to install dependencies:
+First you need to install dependencies, build the project, and run with:
```
Comment (mitas1) [inline README.md]: Too many commas.

Comment (mskrip) [reply README.md]: It's correct in English language

Comment (mskrip) [reply README.md]: It is known as [Oxford comma](http://www.grammarbook.com/punctuation/commas.asp)

**Pull request merged**